protocols/index.js

```diff
@@ -63,13 +65,11 @@ import xonotic from './xonotic.js'
 import altvmp from './altvmp.js'
 import vintagestorymaster from './vintagestorymaster.js'
 import vintagestory from './vintagestory.js'
-import soldat from './soldat.js'
```

Comment:
Nitpick: why did this moved? eslint?

Reply:
Apologies, I had the automated checks state the GAMES_LIST.md needed to be in alphabetical order. The this was just me tidying the alphabetical order in the index.js so that soldat is in alphabetical order for consistency.